.rhcicd/clowdapp-connector-email.yaml

```diff
@@ -201,11 +201,8 @@ parameters:
   description: Maximum number of redelivery attempts (initial call not included)
   value: "2"
 - name: NOTIFICATIONS_CONNECTOR_SEDA_CONCURRENT_CONSUMERS
-  description: Number of concurrent threads processing exchanges with SEDA
-  value: "10"
-- name: NOTIFICATIONS_CONNECTOR_SEDA_QUEUE_SIZE
-  description: Maximum capacity of the SEDA queue
-  value: "10"
+  description: Number of concurrent threads processing exchanges with SEDA; Maximum capacity of the SEDA queue
+  value: "70"
```

Comment:
Could you please detail where 70 comes from?
Because we have 3 pods on prod, it means increasing from 30 to 2100 requests in parallel. It's possible but  did you validated it with BOP team?

Reply:
Not yet, although I think the increase would be 210, instead of 2100

Reply:
yes sorry I added an extra 0 :)